packages/shared/src/utils/constants/contract.tsx

```diff
@@ -12,7 +12,7 @@ export const getLocalizedBasis = () => ({
  * components can be undef or an array containing any of: 'start_date', 'barrier', 'last_digit'
  *     ['duration', 'amount'] are omitted, as they're available in all contract types
  */
-export const getContractTypesConfig = symbol => ({
+export const getContractTypesConfig = (symbol: string) => ({
```

Comment:
Shouldn’t we define here return types for function? for example:
type TContract = {
     title: string;
     ... defining all types for object...
}

export const getContractTypesConfig = (symbol: string): TContract[] => ({
....

Reply:
and also for other functions...